src/librustc_resolve/error_reporting.rs

```diff
@@ -1,856 +0,0 @@
-use std::cmp::Reverse;
```

Comment:
Could you move this renaming into a separate commit, so git doesn't interpret it a `+2,494 −2,494` diff.

Reply:
Okay, I separated commits.